doc/api/cli.md

```diff
@@ -3988,6 +3988,8 @@ documented here:
 
 ### `--harmony-shadow-realm`
 
+### `--heap-snapshot-on-oom`
```

Comment:
Technically, this is not a nodejs flag. But I'm fine documenting it here, as it has value. I do think the docs should state the difference between the heapdump flags that are available.

Reply:
Hi Paul, if you take a closer look this section is called useful v8 flags 😄 https://github.com/nodejs/node/pull/60260/files#diff-7349c17bce4d97526eefcb6f8291d617e7edb92ed9927fd3b9264e8014fc4e7cR3968

> I do think the docs should state the difference between the heapdump flags that are available.

I think that’s a great idea! Do you want to send a follow-up PR for it? You’ve got plenty of expertise in this area.